scripts/HPC_chatbot_preprocessor/parsed_mds/generic/account/account_paragraph_1.txt

```diff
@@ -0,0 +1,36 @@
+Getting an HPC Account
+Getting ready to request an account
+All users of AUGent can request
+an
+account on the HPC, which is part of the Flemish Supercomputing Centre (VSC).
+See HPC policies for more information on who is entitled to an account.
+The VSC, abbreviation of Flemish Supercomputer Centre, is a virtual
+supercomputer centre. It is a partnership between the five Flemish
+associations: the Association KU Leuven, Ghent University Association,
+Brussels University Association, Antwerp University Association and the
+University Colleges-Limburg. The VSC is funded by the Flemish
+Government.
+There are two methods for connecting to HPC-UGent infrastructure:
```

Comment:
so much terminology is introduced here that it is not funny anymore. and that is also not relevant to getting an account. best to write up some separate section explaining what the vsc is, what augent is , what HPC_UGent is within all this. can be in an appendix for all i care